tests/dbshell/test_postgresql.py

```diff
@@ -14,15 +14,16 @@ def _run_it(self, dbinfo):
         That function invokes the runshell command, while mocking
         subprocess.run(). It returns a 2-tuple with:
         - The command line list
-        - The the value of the PGPASSWORD environment variable, or None.
+        - The dictionary of PG* environment variables, or {}.
         """
-        def _mock_subprocess_run(*args, env=os.environ, **kwargs):
+        def _mock_subprocess_run(*args, env=os.environ, **_kwargs):
```

Comment:
Can you revert this change? It seems unrelated.

Reply:
Done